zebra-state/src/request.rs

```diff
@@ -195,4 +195,11 @@ pub enum Request {
     /// Code making this request should apply a timeout layer to the service to
     /// handle missing UTXOs.
     AwaitUtxo(transparent::OutPoint),
+
+    /// Find intersection of provided and best chain and returns block hashes ahead.
+    ///
+    /// Returns
+    ///
+    /// [`Response::Hashes(Vec<block::Hash>)`](Response::Hashes) with max of 500 block hashes.
+    GetHashes(Vec<block::Hash>, Option<block::Hash>),
```

Comment:
Slightly modified from:
https://github.com/ZcashFoundation/zebra/blob/main/zebra-network/src/protocol/internal/request.rs#L81

```suggestion
    FindBlockHashes {
        /// Hashes of known blocks, ordered from highest height to lowest height.
        known_blocks: Vec<block::Hash>,
        /// Optionally, the last block hash to request.
        stop: Option<block::Hash>,
    },
```